### PR TITLE
Fingerprints

### PR DESCRIPTION
**Pull request by marcusellison**

### CODE REVIEW

File: bdr/004-fingerprint-design.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Fingerprint Design
```
Comment (xh3b4sd) [inline bdr/004-fingerprint-design.md:1]: Our latest vocabulary was around `imprint`. I was often thinking about `signature`. Uncertain what the better terminology is.

File: bdr/004-fingerprint-design.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The job of the 'fingerprint' is to score how relevant an update is to other updates.
+
+We want to determine:
+
+- what information we want to capture
+- i.e. context, scale, audience, other relations
+- how to weight this information
+- how the weighting of this information can evolve over time
+- how we can compare the fingerprint of one update to anothe
```
Comment (xh3b4sd) [inline bdr/004-fingerprint-design.md:19]: At our current stage the question might rather be how useful the idea of a fingerprint is while we have to execute on our bootstrapping strategy. So the decision we want to record here might rather be to pursue and include the concept right now or just put it on the back burner.

File: bdr/004-fingerprint-design.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Alternatives
+
+TBD
```
Comment (xh3b4sd) [inline bdr/004-fingerprint-design.md:23]: We actually need some form of identification, it just does not have to be some full fledged fingerprint. In our recent conversations we talked about discovery mechanisms across different dimensions. Maybe it is then just good enough to have a way to jump from graph to graph or update to update.

File: bdr/004-fingerprint-design.md (hunk 1 above)
Comment (marcusellison) [reply bdr/004-fingerprint-design.md:1]: I think signature is better than imprint.

File: bdr/004-fingerprint-design.md (hunk 2 above)
Comment (marcusellison) [reply bdr/004-fingerprint-design.md:19]: I like the idea of changing name to signature and deciding to defer implementation details until later.

File: bdr/004-fingerprint-design.md (hunk 3 above)
Comment (marcusellison) [reply bdr/004-fingerprint-design.md:23]: Agree - a placeholder of sorts

File: bdr/004-fingerprint-design.md (hunk 2 above)
Comment (xh3b4sd) [reply bdr/004-fingerprint-design.md:19]: Let's do it like that.

File: bdr/004-fingerprint-design.md (hunk 3 above)
Comment (xh3b4sd) [reply bdr/004-fingerprint-design.md:23]: Does that mean we want or need to realize the signature idea in some way or what should our decision be here right now?

Comment (marcusellison) [reply bdr/004-fingerprint-design.md:23]: This would be aided by discussion to clarify.

**Comment by xh3b4sd** [comment]
Outdated, closing. Feel free to reopen.

**Pull request closed without merging**